.changeset/great-steaks-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': minor
```

Comment:
Probs doesn't matter a whole bunch but this feels more like a patch change to fix the accessibility rather than a minor update that's usually for adding new capabilities